kinta-lib/src/main/kotlin/com/dailymotion/kinta/integration/appgallery/UploadData.kt

```diff
@@ -0,0 +1,3 @@
+package com.dailymotion.kinta.integration.appgallery
+
+data class UploadData(val uploadUrl: String, val authCode: String)
```

Comment:
Does this need to be a data class? Data classes have a cost and cause some compatibilities issues: https://jakewharton.com/public-api-challenges-in-kotlin/

Unless there's a strong reason, I would keep this a regular class

Reply:
👍